icon.c

```diff
@@ -112,7 +112,15 @@ static char *resolve_icon(struct mako_notification *notif) {
 			errno = 0;
 			int32_t icon_size = strtol(relative_path, NULL, 10);
 			if (errno || icon_size == 0) {
-				continue;
+				errno = 0;
```

Comment:
Can you add a comment here explaining what this is looking for?

Reply:
Done.